### PR TITLE
'build.sh' script that triggers a framework build

### PR DESCRIPTION
**Pull request by cvasilak**
a shell script that triggers a framework build. Have updated the readme.md with instructions on how to use it.

(It's the output of this, that I used on HelloWorld and Contacts demos)

@corinnekrych @matzew mind to try?

### CODE REVIEW

**Comment by lholmquist** [comment]
:+1:   built good, followed the directions to import into a project,  and then "import"ed the Library and it built ok.

didn't try using it to send push messages though

**Comment by corinnekrych** [comment]
:eyes:

File: README.md [hunk 1]
```diff
@@ -49,6 +51,20 @@ git submodule add https://github.com/aerogear/aerogear-ios-push.git
...
+
+To build the library simply run the build script:
+
+    build.sh
+
+The build script will generate a ```build``` folder in the current directory, containing an universal framework library that you can drag-and-drop in your project. The framework build resides in:
+
+    build/Release-universal/AeroGearPush.framework
+
+Now, on your application targets’ “General” settings tab and in the “Linked Frameworks and Libraries” section, drag-and-drop the the framework build.  Then click the 'Build Phases' tab, click the '+' symbol, select "New Copy Files Phase", expand the section, select 'Frameworks' from the Destination combo box.  Click the '+' symbol and select the 'AeroGearPush.framework'
```
Comment (corinnekrych) [inline README.md]: Duplicate the in "drag-and-drop the the framework build"

**Comment by matzew** [comment]
```
** BUILD SUCCEEDED **
```

:+1:

**Comment by corinnekrych** [comment]
+1 it will be a nice start for @vibe13 

ready to merge

**Comment by cvasilak** [comment]
:rocket: landed on https://github.com/aerogear/aerogear-ios-push/commit/0ce14b44d89f0a95688a4068aed4767716b4b40b

**Pull request merged**